lib/modules/apostrophe-images/lib/cursor.js

```diff
@@ -40,6 +40,79 @@ module.exports = {
       }
     });
 
+    // Filter. Get images depending of the
+    // orientation of their attachments. This orientation
+    // is computed during import, comparing width and height
+
+    self.addFilter('orientation', {
+      def: null,
+      finalize: function() {
+        const orientation = self.get('orientation');
+        if (!orientation) {
+          return;
+        }
+
+        self.and(getCriteria(orientation));
+
+        function getCriteria(orientation) {
+          if (orientation === 'landscape') {
+            return {
+              'attachment.landscape': true
+            };
+          } else if (orientation === 'portrait') {
+            return {
+              'attachment.portrait': true
+            };
+          } else {
+            // Last case is square
+            return {
+              'attachment.square': true
+            };
+          }
+
+        }
+      },
+      safeFor: 'public',
+      launder: function(s) {
+        return self.apos.launder.string(s);
+      },
+      choices: function(callback) {
+        const { choices } = options.module.filters
+          .find((filter) => filter.name === 'orientation');
+
+        return callback(null, choices);
+      }
+    });
+
+    // Filter. Get images depending of their
+    // type, the available images extensions come from
+    // self.fileGroups in apostrophe-attachments
```

Comment:
End with a period please. `self.fileGroups` and `apostrophe-attachments` should be in back-ticks for code styling.

Reply:
Done.